src/base/rss/rss_autodownloader.cpp

```diff
@@ -245,6 +245,7 @@ void AutoDownloader::processJob(const QSharedPointer<ProcessingJob> &job)
         }
 
         rule.setLastMatch(job->articleDate);
+        rule.matched(job->articleTitle);
```

Comment:
The code can't go in matches as that is called from many places, and we have to know when a match caused a download, so we need a way for the downloader to tell the rule it has matched an episode and it will be downloaded so the rule can add that article to the previously matched episodes.

What's a more appropriate name, articleMatched? ruleMatched?